LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2003-present, All Licensees
```

Comment:
I pulled `2003` from the earliest commit on the project, let me know if this is incorrect.